tests/test_custom_model.py

```diff
@@ -0,0 +1,26 @@
+from datetime import datetime, timezone
```

Comment:
Not sure what else to test there. It basically just makes a http call and parses the response. Open to suggestions though.

Reply:
It would test the HTTP call. We have these tests for most of the HTTP calls. For example if the response came back without one of those parameters this test would catch that.